.gitignore

```diff
@@ -31,6 +31,9 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+Pipfile
+Pipfile.lock
+pyproject.toml
```

Comment:
If we are gonna add this to every module we should add them in the `cookiecutter-invenio-module` so they are there from the initial commit.

Reply:
Sure... created here: https://github.com/inveniosoftware/cookiecutter-invenio-module/pull/145